.env.example

```diff
@@ -27,4 +27,6 @@ NATIVE_ASSET_SYMBOL = ""
 REWARDS_ASSET_SYMBOL = ""
 
 # Address of the rewards token. Leave empty to deploy system without rewards token
-REWARDS_ASSET_TOKEN = ""
\ No newline at end of file
+REWARDS_ASSET_TOKEN = ""
+
+IS_COVERAGE =  "true"
```

Comment:
This variable must be `true` only to start **coverage**. When running normal tests, it should be `false`